workspaces/kiali/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@internal/kiali",
+  "version": "1.0.0",
+  "private": true,
+  "engines": {
+    "node": "18 || 20"
+  },
+  "scripts": {
+    "tsc": "tsc",
+    "tsc:full": "tsc --skipLibCheck true --incremental false",
+    "build:all": "backstage-cli repo build --all",
+    "build:api-reports": "yarn build:api-reports:only",
+    "build:api-reports:only": "backstage-repo-tools api-reports --allow-all-warnings -o ae-wrong-input-file-type,ae-undocumented --validate-release-tags",
+    "clean": "backstage-cli repo clean",
+    "test": "backstage-cli repo test",
+    "test:all": "backstage-cli repo test --coverage",
+    "fix": "backstage-cli repo fix",
+    "lint": "backstage-cli repo lint --since origin/main",
+    "lint:all": "backstage-cli repo lint",
+    "prettier:check": "prettier --check .",
+    "new": "backstage-cli new --scope @backstage-community",
+    "postinstall": "cd ../../ && yarn install"
+  },
+  "workspaces": {
+    "packages": [
+      "packages/*",
+      "plugins/*"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/community-plugins",
+    "directory": "workspaces/kiali"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.28.0",
+    "@backstage/e2e-test-utils": "^0.1.1",
+    "@backstage/repo-tools": "^0.8.0",
```

Comment:
```suggestion
    "@backstage/repo-tools": "^0.10.0",
```
Update this and then run `yarn build:api-reports` and commit the changes. That should get the CI green

Reply:
After update ....

```bash
agutierr@agutierr-thinkpadp1gen3:~/Projects/backstage/community-plugins/workspaces/kiali (migrate-kiali)$ yarn build:api-reports

TIP: You can generate api-reports for select packages by passing package paths:

       yarn build:api-reports packages/config packages/core-plugin-api plugins/*

# Generating package API reports
## Processing plugins/kiali-backend

Error: Unable to load file: /home/agutierr/Projects/backstage/community-plugins/workspaces/kiali/dist-types/plugins/kiali-backend/src/index.d.ts

```
This is weird due plugins are under **kiali** folder not **dist-types** and there is not **index.d.ts**

Reply:
I did this on my local without any issues, this is very odd.